### DISCUSSION TITLE
Self hosted firefly and csv importer - setup question

### DISCUSSION THREAD

**Discussion by dvdrip1** [firefly-iii/firefly-iii#4767] [Firefly III] [answered]
Hi,

I have firefly set up succesfully on a pi 4, using the LAMP-installation (was running this setup anyway so didn't use Docker).
However I am having trouble to get csv importer up with firefly.

My pi is running at 192.168.178.210 in my home network. I have created two virtual hosts in Apache 2, one vhost listening to *:80 with alias banking/ (this is firefly) and the other vhost *:80 with alias csv-import. My directories are /var/www/banking/firefly-iii and /var/www/banking/csv-importer.

I can reach both "sites" on another computer perfectly (as I edited the hosts-file on that PC). 

However, what is the firefly-serveraddress I should use to configure csv-importer? When I use http://192.168.178.210 it will not work as that is my default (other) site on the webserver and http://banking/ is not recognised.

**Comment by nebula2** [comment, marked as answer]
they are running on fhe same machine right?
specifying the port could help.

to banking/ cannot be resolved: may it be that you need to specify a host entry if your dns does not resolve this?

**Reply by dvdrip1** [reply]
well, http://banking/ is not known on the network, it's working since I've edited the hosts file on my desktop, and since the VirtualHost actually does listen to this name, the site works from my Desktop.

I have found out I can bind multiple IP's to my eth0 address, or actually wlan0, so the Pi has multiple IP's to listen to. Each site now listens to a separate IP, although I still need to find out how to bind those IP fixed; when restarting the Pi they will not be bound anymore (I'm using ifconfig from the cmd line)

**Reply by nebula2** [reply]
maybe [this](https://raspberrypi.stackexchange.com/questions/37920/how-do-i-set-up-networking-wifi-static-ip-address-on-raspbian-raspberry-pi-os) helps you out?

**Reply by dvdrip1** [reply]
Thanks for sending this link. Through that page and some key words, I found [another page](https://www.raspberrypi.org/forums/viewtopic.php?t=191657), post of hippy helped me out (the code added to /etc/network/interfaces). This creates a few IP aliases, so each site can listen to its own IP on the network. 

Thanks for the pointers, nebula2!
